spec/models/spree/order_spec.rb

```diff
@@ -1544,4 +1544,50 @@ def advance_to_delivery_state(order)
       expect(order.voucher_adjustments).to eq(expected_adjustments)
     end
   end
+
+  describe '#applied_voucher_rate' do
```

Comment:
For future reference: it's better to include the spec in the same commit as the code change needing the spec.